README.md

```diff
@@ -2,16 +2,216 @@
 
 #### Table of Contents
 
-1. [Overview](#overview)
-2. [Module Description - What the module does and why it is useful](#module-description)
-3. [Setup - The basics of getting started with iis](#setup)
-    * [Setup requirements](#setup-requirements)
-    * [Beginning with iis](#beginning-with-iis)
-4. [Usage - Configuration options and additional functionality](#usage)
-5. [Reference - An under-the-hood peek at what the module is doing and how](#reference)
+1. [Description](#description)
+2. [Setup - The basics of getting started with IIS](#setup)
+3. [Usage - Configuration options and additional functionality](#usage)
+4. [Reference - An under-the-hood peek at what the module is doing and how](#reference)
+    * [Facts](#facts)
+    * [Types/Providers](#types/providers)
+        * [iis_application_pool](#iis_application_pool)
+        * [iis_site](#iis_site)
 5. [Limitations - OS compatibility, etc.](#limitations)
 6. [Development - Guide for contributing to the module](#development)
 
-## Overview
+## Description
 
-This module adds a new provider capable of executing iis commands.
+This module adds a provider to manage IIS sites and application pools.
+
+## Setup
+
+### Beginning with puppetlabs-iis
+
+This module requires that IIS is already installed on your server. This could be done using the [puppet/windowsfeature module](https://forge.puppet.com/puppet/windowsfeature) from Vox Pupuli by ensuring the `Web-WebServer` & `Web-Scripting-Tools` Windows Features are present.
+
+**Note:** This module requires Powershell 3, which is not the default for Windows Server 2008R2 and will need to be installed for this module to function.
```

Comment:
We are officially going with PowerShell v2 as minimum. Change this line to reflect this and I will merge.